modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -98,6 +98,10 @@ Creating a machine pool with taints is only possible if the cluster already has
 Alternatively, you can add the node labels and taints after you create the machine pool.
 ====
 
+ifdef::openshift-rosa[]
+. Optional: Select additional custom security groups to use for nodes in this machine pool. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for _Security groups_ under _Additional resources_.
```

Comment:
@laubai It seems confusing as we are talking about nodes in a machine pool. The machine pool itself called as node pool or nodes in this case. Suggestion is 
`Optional: Select additional custom security groups to use in this machine pool. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups after you create the machine pool. For more information, see the requirements for _Security groups_ under _Additional resources_.`